### PR TITLE
Add Move and PhasedMove effects.

### PR DESCRIPTION
**Pull request by bartmate**
I added the effect which corresponds to a move in quantum chess.

### CODE REVIEW

File: unitary/alpha/qubit_effects_test.py [hunk 1]
```diff
@@ -34,6 +34,36 @@ def test_superposition():
...
+    piece2 = alpha.QuantumObject("b", 0)
+    board.add_object(piece1)
+    board.add_object(piece2)
+    alpha.Move()(piece1, piece2)
+    expected_circuit = cirq.Circuit()
+    a = cirq.NamedQubit("a")
+    b = cirq.NamedQubit("b")
+    expected_circuit.append(cirq.X(a))
+    expected_circuit.append(cirq.SWAP(a, b))
+    assert board.circuit == expected_circuit
```
Comment (dstrain115) [inline unitary/alpha/qubit_effects_test.py:49]: Can you also test expected results too?  I hit this when my circuit was wrong before.

For instance,

```
results = board.peek([piece1, piece2], count =100)
assert all(result == [1,0] for result in results)
```

Same for below.

**Review by dstrain115 — APPROVED** [review]
Looks good.  Recommend extending the test slightly.

Thanks for contributing and congratulations on being the first new contributor!!!

File: unitary/alpha/qubit_effects_test.py (hunk 1 above)
Comment (bartmate) [reply unitary/alpha/qubit_effects_test.py:51]: I added two tests of `results` for both operations: One before the operation and one after the operation.

Comment (bartmate) [reply unitary/alpha/qubit_effects_test.py:51]: What is the next step?

Comment (dstrain115) [reply unitary/alpha/qubit_effects_test.py:51]: Next step is to merge it into main, which I did for you.  Thanks for contributing!

**Pull request merged**